public/index.html

```diff
@@ -3,27 +3,56 @@
 	<head>
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<link rel="icon" href="favicon.ico">
+		<link rel="icon" href="assets/image/favicon.ico">
 
 		<title>OpenFisca - Turn law into software</title>
 
-		<link rel="stylesheet" type="text/css" href="css/main.css">
+		<link rel="stylesheet" type="text/css" href="assets/css/main.css">
 	</head>
 	<body>
-		<img id="logo" src="logo-openfisca.svg" alt="OpenFisca">
-
-		<h1>Turn law into software</h1>
-
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
-
-		<h3>Calculate citizens rights</h3>
-		<p>From an individual situation, calculate any variable of a national tax and benefit system.</p>
-
-		<h3>Predict company taxes</h3>
-		<p>Model any kind of entity and evaluate their eligibility to fiscal obligations and benefits.</p>
-
-		<h3>Assess the impact reforms could have</h3>
-		<p>Compare law as it is now and as it could be.</p>
+		<header>
+			<img id="logo" src="assets/image/logo-openfisca.svg" alt="OpenFisca">
+
+			<h1>Turn law into software</h1>
+
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		</header>
+
+		<section>
+			<div>
+				<h3>Calculate citizens' rights</h3>
+				<p>From an individual situation, calculate any variable of a national tax and benefit system.</p>
+			</div>
+
+			<figure>
+				<video class="screenshot" src="assets/video/mes-aides.mp4" poster="assets/image/mes-aides.png" controls></video>
+				<figcaption><a href="https://beta.gouv.fr/startup/mes-aides">Mes Aides</a> uses OpenFisca and its Web API to inform French citizens on their eligibility to national and local benefits.</figcaption>
+			</figure>
+		</section>
+
+		<section>
+			<div>
+				<h3>Assess company taxes</h3>
+				<p>Model any kind of entity and evaluate its eligibility to fiscal obligations and benefits.</p>
+			</div>
+
+			<figure>
+				<video class="screenshot" src="assets/video/embauche.mp4" controls></video>
+				<figcaption><a href="https://embauche.beta.gouv.fr/">Embauche</a> uses OpenFisca and a specific extension to allow employers and accountants to assess the hiring cost a new employee, including fiscal deductions.</figcaption>
+			</figure>
+		</section>
+
+		<section>
+			<div>
+				<h3>Evaluate reforms impact</h3>
+				<p>Compare the law now to what it could be.</p>
+			</div>
+
+			<figure>
+				<img class="screenshot" src="assets/image/revenu-de-base.jpg" alt="Comment appréhender le coût d’un revenu de base ?">
+				<figcaption><a href="http://www.revenudebase.info/2017/04/07/apprehender-cout-dun-revenu-de-base/">This publication (in french)</a> of the French Movement for a Basic Income uses OpenFisca to evaluate the cost and benefits of a reform introducing a basic income in France.</figcaption>
```

Comment:
There is no documentation about a "Python API". As its specific to a country package, I don't know how it could be described but for this case I added [this reference](https://github.com/openfisca/openfisca-france#a-installation-minimale-pip-install).